python/higgstotautauanalysiswrapper.py

```diff
@@ -4,61 +4,437 @@
+		                                 help="Add paths to environment variable LD_LIBRARY_PATH.")
+		runningOptionsGroup.add_argument("--profile", default="",
+		                                 help="Measure performance with profiler. Choose igprof or valgrind.")
+		runningOptionsGroup.add_argument("--profile-options", default="pp",
+		                                 help="Additional options for profiling. Choose memory (mp) or performance (pp). [Default: %(default)s]")
+		runningOptionsGroup.add_argument("-r", "--root", default=False, action="store_true",
+		                                 help="Open output file in ROOT TBrowser after completion.")
+		runningOptionsGroup.add_argument("-b", "--batch", default=False, const="naf", nargs="?",
+		                                 help="Run with grid-control. Optionally select backend. [Default: %(default)s]")
+		runningOptionsGroup.add_argument("--pilot-job-files", "--pilot-jobs", default=None, const=1, type=int, nargs="?",
+		                                 help="Number of files per sample to be submitted as pilot jobs. [Default: all/1]")
+		runningOptionsGroup.add_argument("--files-per-job", type=int, default=15,
+		                                 help="Files per batch job. [Default: %(default)s]")
+		runningOptionsGroup.add_argument("--area-files", default=None,
+		                                 help="Additional area files. [Default: %(default)s]")
+		runningOptionsGroup.add_argument("--wall-time", default="24:00:00",
+		                                 help="Wall time of batch jobs. [Default: %(default)s]")
+		runningOptionsGroup.add_argument("--memory", type=int, default=3000,
+		                                 help="Memory (in MB) for batch jobs. [Default: %(default)s]")
+		runningOptionsGroup.add_argument("--cmdargs", default="-cG -m 3",
+		                                 help="Command line arguments for go.py. [Default: %(default)s]")
+		runningOptionsGroup.add_argument("--se-path",
+		                                 help="Custom SE path, if it should different from the work directory.")
+		runningOptionsGroup.add_argument("--log-to-se", default=False, action="store_true",
+		                                 help="Write logfile in batch mode directly to SE. Does not work with remote batch system")
+		runningOptionsGroup.add_argument("--partition-lfn-modifier", default = None,
+		                                 help="Forces a certain access to input files. See base conf for corresponding dictionary")
+		
+	
+	def import_analysis_configs(self):
+		# define known analysis keys here
+		analysis_configs_dict = {
+			#'SM' : 'HiggsAnalysis.KITHiggsToTauTau.data.ArtusConfigs.Run2Analysis_base', #TODO
+			#'sm' : 'HiggsAnalysis.KITHiggsToTauTau.data.ArtusConfigs.Run2Analysis_base',
+			'MSSM' : 'HiggsAnalysis.KITHiggsToTauTau.data.ArtusConfigs.Run2MSSM_base',
+			'mssm' : 'HiggsAnalysis.KITHiggsToTauTau.data.ArtusConfigs.Run2MSSM_base'
```

Comment:
At some point we should agree on whether we want to keep it like this (i.e. enter all analyses/configs twice) or agree on some other kind of fuzzy matching for the names.